calcom/cal.com
# chore: switch over to use variable for app/companyName

* enable_apps_description - appName
* recording_from_your_recent_call - appName
* the_calcom_team - companyName
* calcom_is_better_with_team - appName

## What does this PR do?

Uses variables in a few more of the i18n files, to show the App/Company name in more places on self-hosted.

## Type of change

- Chore (refactoring code, technical debt, workflow improvements)
- New feature (non-breaking change which adds functionality)

## How should this be tested?

See the new variables in place, when an APP_NAME or COMPANY_NAME is set, otherwise it still falls back to "Cal.com"

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
